pallets/api/src/fungibles/mod.rs

```diff
@@ -93,6 +95,33 @@ pub mod pallet {
 	#[pallet::pallet]
 	pub struct Pallet<T>(_);
 
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {
```

Comment:
Missing doc comments on the the event type itself.

Reply:
See commit `refactor: incompatible event based on standard`